lib/testing-helpcenter/main_page_links/help_center_contribution.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# /guides/become-translator.aspx
+# https://user-images.githubusercontent.com/40513035/120451570-697f2780-c346-11eb-9fbd-577b0b625a3d.png
+
+module TestingHelpCentreOnlyoffice
+  class HelpCenterContribution
+    include PageObject
+
+    div(:participating_block, xpath: "//div[@id='Participating_block']")
+
+    def initialize(instance)
+      super(instance.webdriver.driver)
+      instance.webdriver.wait_until { participating_in_translation_block_present? }
+    end
+
+    def participating_in_translation_block_present?
+      participating_block_element.present?
```

Comment:
Maybe ditch page object all-together in this file and just left
```
def participating_in_translation_block_present?
 @instance.webdriver.element_present?("//div[@id='Participating_block']")
end
```

Reply:
Why?
I think it's better if code is written in one pattern using page-object.

Reply:
In this specific case PageObject has no advantages, but if using one pattern is more important that shorter classes - leaving this way